games/f.yaml

```
@@ -176,6 +176,30 @@
   url: http://www.newbreedsoftware.com/fop/
   info: A Dandy clone with Gauntlet influences. Choose from 8 classes and roam the maze taking treasure, vanquishing monters, and finding the exit. Use explosives against enemies and keys on locked doors.
 
+- name: Fish Fillets NG
+  originals:
+  - Fish Fillets
+  type: remake
+  repo: 'https://sourceforge.net/projects/fillets/'
+  url: 'http://fillets.sourceforge.net/'
+  development: complete
+  status: playable
+  content: free
```

```suggestion
  content: open
```
I read that the content is also GPL; note OSGC uses "free" as in "beer"

---

Thanks, fixed.